### ISSUE TITLE
Puter Terminal does not accept mobile input

### ISSUE THREAD

**Issue by Jeklah** [HeyPuter/phoenix#18]
I tried using the terminal on my mobile but it did not appear to recognise any keystrokes, apart from enter strangely enough. See attached video.

Uploading screen-20240215-132240.mp4…

Uploading screen-20240215-132240.mp4…

**Comment by Jeklah** [comment]
https://youtube.com/shorts/x1d9PGj1Vb4?si=tPZc-9rBY5SfhvK0

**Comment by jelveh** [comment]
Hi @Jeklah, thanks for reporting this. This is a problem with xterm.js, we're gonna look into a workaround.

**Comment by jelveh** [comment]
@AtkinsSJ @KernelDeimos could you look into this?

**Comment by AtkinsSJ** [comment]
:thinking: I get this problem in Chrome but not in Firefox.

**Comment by Jeklah** [comment]
I just tried testing this in Firefox, the website came up but I was unable to login...is it down temporarily at the moment? I couldn't even get onto the website on Chrome.

One difference I did notice, whether it'll make the difference between the issue being seen on Chrome and not FF I'm not sure, but FF opened up puter.com with https while Chrome did not.

Will check again later to see if site is back up and will check both browsers.

**Comment by jelveh** [comment]
@Jeklah Yes unfortunately we were down. I just tried Chrome, Safari, and FF on iOS and they all worked.

@AtkinsSJ shall we... fix xterm.js? :)

**Comment by AtkinsSJ** [comment]
> @AtkinsSJ shall we... fix xterm.js? :)

I'll add that to my list! 🦬➕
